src/MassTransit/Configuration/Configuration/IReceiveEndpointConfiguration.cs

```diff
@@ -32,7 +32,12 @@ public interface IReceiveEndpointConfiguration :
         /// <summary>
         /// Completed once the receive endpoint dependencies are ready
         /// </summary>
-        Task Dependencies { get; }
+        Task DependenciesStarted { get; }
```

Comment:
Can we just agree on a single term? Dependencies or Dependents.

Reply:
they are completely different things (well at least for me :-) )
A Dependency of B
B Dependents of A

Reply:
`DependenciesReady` - consistent with IAgent